### PR TITLE
Show Azure storage container region (WOR-740).

### PR DESCRIPTION
**Pull request by cahrens**
At some point, the region was added to the `controlledResourceMetadata`.

image

### CODE REVIEW

File: src/pages/workspaces/workspace/Dashboard.js [hunk 1]
```diff
@@ -534,9 +535,9 @@ const WorkspaceDashboard = _.flow(
         h(InfoRow, { title: 'Cloud Name' }, [
           h(AzureLogo, { title: 'Microsoft Azure', role: 'img', style: { height: 16 } })
         ]),
-        // h(InfoRow, { title: 'Location' }, [
-        //   h(TooltipCell, !!storageLocation ? [getRegionFlag(storageLocation), ' ', getRegionLabel(storageLocation)] : ['Loading'])
-        // ]), depends on TOAZ-265
+        h(InfoRow, { title: 'Location' }, [
+          h(TooltipCell, !!storageLocation ? [getRegionFlag(storageLocation), ' ', getRegionLabel(storageLocation)] : ['Loading'])
```
Comment (cahrens) [inline src/pages/workspaces/workspace/Dashboard.js:539]: `getRegionFlag` and `getRegionLabel` currently only supports useast. @mlilynolting are you adding more regions to that structure?

image

**Review by marctalbott — APPROVED** [review]

File: src/pages/workspaces/workspace/Dashboard.js (hunk 1 above)
Comment (cahrens) [reply src/pages/workspaces/workspace/Dashboard.js:539]: If that isn't happening very soon, I am inclined to show the raw region value here if it isn't found in the region structure in `azure-utils`.

Comment (mlilynolting) [reply src/pages/workspaces/workspace/Dashboard.js:539]: Yes, I'm adding those regions. I have no issue with falling back on the raw region value for display as of this PR.

**Review by mlilynolting — APPROVED** [review]
lgtm!

**Review by mlilynolting — APPROVED** [review]

**Comment by cahrens** [comment]
Integration test failures (IA tests) are unrelated. I added the list of regions from https://github.com/DataBiosphere/terra-ui/pull/3724.

**Pull request merged**